src/blocks/FilterBlock/FilterBlock.tsx

```diff
@@ -78,7 +73,13 @@ const FilterBlock: React.FC<FilterBlockProps> = ({
                 </Row>
             )}
             <Row className={b('block-container')}>
-                <ConstructorBlocks items={container} />
+                <CardLayoutBlock title="" colSizes={colSizes}>
```

Comment:
Why is `title` empty?

Reply:
> Why is `title` empty?

it was same before, didn't change anything here

Reply:
Can you make `title` as not necessary in CardLayoutBlock? It looks not so good